habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -589,6 +589,20 @@ class MarkerRelPosSensorConfig(LabSensorConfig):
     type: str = "MarkerRelPosSensor"
 
 
+@dataclass
+class HandleBBoxSensorConfig(LabSensorConfig):
+    r"""
+    Articulation task only. Returns the bounding box of the handle in pixels
+    """
+    type: str = "HandleBBoxSensor"
+    height: int = 480
+    width: int = 640
+    # The pixel size (in cell) drawn in the image of reach handle point
+    pixel_size: int = 2
```

Comment:
I don't quite understand this comment. Is this the pixel size of the border of the bbox?

Reply:
update the comment, thanks!